ui/lib/pki/addon/routes/keys/create.js

```diff
@@ -1,3 +1,18 @@
-import Route from '@ember/routing/route';
+import PkiKeysIndexRoute from '.';
+import { inject as service } from '@ember/service';
+import { withConfirmLeave } from 'core/decorators/confirm-leave';
 
-export default class PkiKeysCreateRoute extends Route {}
+@withConfirmLeave()
```

Comment:
🎉

Reply:
yes! such a nice decorator, thank you for making!